examples/api/test/cupertino/list_tile/cupertino_list_tile.0_test.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/cupertino/list_tile/cupertino_list_tile.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Test CupertinoListTile respects properties', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('CupertinoListTile respects properties', (WidgetTester tester) async {
```

Since this is a test, we can omit this word.